chart/templates/server-ide-configmap.yaml

```diff
@@ -104,6 +104,34 @@ options:
 
 defaultIde: "code"
 defaultDesktopIde: "code-desktop"
+
+clients:
+  vscode:
+    defaultDesktopIDE: "code-desktop"
+    desktopIDEs: ["code-desktop"]
+    installationSteps: [
+      "Click <b>Allow</b> in the dialog by your browser.",
+      "If you don't see the dialog, make sure you have <a target='_blank' class='gp-link' href='https://code.visualstudio.com/download'>VS Code</a> installed on your machine.",
```

Comment:
**nitpick**: Couldn't find where to comment about this and it's probably out of the scope of the changes in this PR but it could be more clear and useful now to change the title here to the suggestion from https://github.com/gitpod-io/gitpod/pull/6270#discussion_r734677553 or make the action more clear and separate it from the state.

Reply:
Ok change phase to `Running` and added `Opening Workspace...` as status description. I also replaced `Opening IDE...` with `Opening Workspace...`. I don't think everybody know what IDE stands for.

Reply:
I'd expect the _Opening Workspace ..._ heading to be closer to the instructions and below the workspace entry as seen in the `AFTER` screenshot in https://github.com/gitpod-io/gitpod/pull/7711#pullrequestreview-859556119, since this heading is more related to the actions than the actual lifecycle state of the workspace.

However, I don't think the current approach is bad to hold this back for now. We can always discuss this in depth in https://github.com/gitpod-io/gitpod/issues/6602. Cc @loujaybee